neural_transport/models/flowmatching.py

```diff
@@ -161,7 +159,8 @@ def inference_forward(self, batch):
         )
 
         # solve the ODE to get the trajectory
-        sol = self.solver.sample(time_grid=time_grid,
+        solver = ODESolver(velocity_model=velocity_model)
```

Comment:
fair! It is probably fine to instantiate a new solver during every inference step